src/promptflow/promptflow/_core/tool.py

```diff
@@ -160,7 +160,7 @@ def __post_init__(self, function, input_mapping):
         self._func_obj, self.func_path = _get_function_path(function)
         self._input_mapping = input_mapping or {}
         dynamic_list_func_inputs, _, _, _ = function_to_interface(
-            self._func_obj, gen_custom_type_conn=True, skip_prompt_template=True
+            self._func_obj, gen_custom_type_conn=True, skip_prompt_template=True, add_ui_hints=False
```

Comment:
Add comment here to explain why add_ui_hints needs to be False

Reply:
Added, thanks.